### PR TITLE
Added code for the program to convert given no to word in general question

### PR DESCRIPTION
**Pull request by harsh6768-svg**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/smv1999/CompetitiveProgrammingQuestionBank/blob/master/CONTRIBUTING.md)?

Yes

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests

#861 
The program to convert number in word format 

## Fill up the DCP form 

If you are a part of DCP 21, fill up this [form](https://docs.google.com/forms/d/e/1FAIpQLSeRAJnHE4eAa_W0IUeNZgheiy5POX3I3tQB7DtFsiB13D5w2g/viewform) after the PR is merged.

### The links provided in the form will be checked thoroughly, if someone tries fooling around, strict actions will be taken against the individual.

### CODE REVIEW

File: General Questions/possible no from word.cpp [hunk 1]
```diff
@@ -0,0 +1,118 @@
+/* C program to print a given number in words. The program
+handles numbers from 0 to 9999 
+Just take an example  if the user privide the number
+1234 -Input
+One thousand Two hundred and twenty four -Output
+
```
Comment (noviicee) [inline General Questions/possible no from word.cpp:6]: Remove the extra blank lines.

File: General Questions/possible no from word.cpp [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+					printf("%s ",
+						single_digits[*num - '0']);
+			}
+		}
+		++num;
+	}
+}
+
+/* Driver program to test above function */
+int main(void)
```
Comment (noviicee) [inline General Questions/possible no from word.cpp:110]: Take dynamic input.

File: General Questions/possible no from word.cpp [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+{
+	convert_to_words("9923");
+	convert_to_words("523");
+	convert_to_words("89");
+	convert_to_words("8");
+
+	return 0;
+}
```
Comment (noviicee) [inline General Questions/possible no from word.cpp:118]: Add space-time complexity.

**Review by noviicee — CHANGES_REQUESTED** [review]

File: General Questions/possible no from word.cpp (hunk 3 above)
Comment (harsh6768-svg) [reply General Questions/possible no from word.cpp:132]: @noviicee every thing done time complexity added in the last part made the code dynamic with no of test cases and spaces removed please check

File: General Questions/possible no from word.cpp [hunk 4]
```diff
@@ -0,0 +1,142 @@
+/* C program to print a given number in words. The program
```
Comment (noviicee) [inline General Questions/possible no from word.cpp:1]: It's a C++ program right?

Comment (harsh6768-svg) [reply General Questions/possible no from word.cpp]: sorry @novicee updated my mistake

**Review by noviicee — COMMENTED** [review]
The program is giving runtime errors when I'm trying to execute it. Please check if everything is correct.

**Comment by harsh6768-svg** [comment]
now check @noviicee

**Review by noviicee — APPROVED** [review]

**Pull request merged**